src/App.vue

```diff
@@ -2,32 +2,47 @@
   <div id="app">
     <div
       ref="wrapper"
-      class="min-h-screen wrapper"
-    >
+      class="min-h-screen wrapper">
+      <div
+        v-if="alert"
+        style="text-align: center;line-height: 5rem;">
+        Unsupported browser.
+      </div>
       <router-view />
     </div>
   </div>
 </template>
 
 <script>
 import { mapActions, mapGetters } from 'vuex';
+import { detect } from 'detect-browser';
 import aeternity from './utils/aeternity';
 import { wallet } from './utils/walletSearch';
 import Backend from './utils/backend';
 import { EventBus } from './utils/eventBus';
-import Util from './utils/util';
+import Util, { IS_MOBILE_DEVICE } from './utils/util';
 
 export default {
   name: 'App',
   data() {
     return {
       page: 1,
+      browser: detect(),
+      alert: false,
     };
   },
   computed: {
     ...mapGetters(['settings', 'tipSortBy']),
+    supportedBrowsers() {
+      return ['chrome', 'firefox'];
```

Comment:
i think all chromium browsers should be supported (i.e. Opera, Brave, even New Edge),

Reply:
You are generally correct but we haven't actually tested the extension on Opera and the New Edge, so its better to warn people than to mislead them at this point.

Reply:
I actually use mainly Opera (among the others), it works perfectly. Tough on the chrome-Edge the site works, you just cant install chrome extensions, so not sure what "unsupported should mean".

Reply:
btw in create-profile help it is mentioned Brave and Opera
![image](https://user-images.githubusercontent.com/632282/78704238-385ac100-7914-11ea-90b7-adfe7e49e628.png)
So I guess, showing "unsupported browser" might be awkward